app/assets/stylesheets/templates/_admin.scss

```diff
@@ -0,0 +1,44 @@
+.admin {
+
+  div.header {
+
+    * {
+      margin: 0;
+      margin-bottom: 1.0rem;
+    }
+
+    h1 {
+      margin-bottom: 1.0rem;
+    }
+
+    h2.with-link {
+
+      a {
+        color: $color-purple;
+        font-size: 1.9rem;
+        font-weight: bold;
+        &:hover {
+          color: #8E8EEF;
+        }
+      }
+    }
+
+    p.text--smaller {
+      font-size: 1.2rem;
+      max-width: none;
+    }
+  }
+
+  div.info-details {
+    margin-top: 6.0rem;
+
+    .h3 {
+      margin-bottom: 1.0rem;
+    }
+  }
```

Comment:
usually it's best to avoid tag selectors and stick with classes. so for example, `.info-details` instead of `div.info-details`.